docker/upload.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+import os
+import time
+import json
+
+images = ["ubuntu", "graphiteapp/graphite-statsd", "g4s8/artipie-base"]
+
+# Start artipie with preconfigured docker repo
+def start_artipie():
+    print("Starting artipie")
+    artipie_yml = """
+meta:
+  storage:
+    type: fs
+    path: /var/artipie/configs
+  layout: flat
+"""
+    my_docker = """
+repo:
+  type: docker
+  storage:
+    type: fs
+    path: /var/artipie/data
+  permissions:
+    "*":
+      - "*"
+"""
```

Comment:
@Sammers21 these variables seems to be constants, and could be moved out of function's scope

Reply:
@g4s8, why?